src/plugins/discover/public/application/_discover.scss

```diff
@@ -6,17 +6,69 @@
 }
 
 .dscAppContainer {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  display: flex;
+  flex-direction: column;
   > * {
     position: relative;
   }
 }
+
 discover-app {
   flex-grow: 1;
 }
 
+.dscHitsCounter,
+.dscTimeChartHeader {
+  flex-grow: 0;
+}
+
+.dscApp {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  overflow: hidden;
+}
+
+.dscApp__header {
+  border-bottom: $euiBorderThin;
+  margin-bottom: $euiSize;
+}
+
+.dscApp__frame {
+  position: relative;
+  flex-grow: 1;
+  display: flex;
+  overflow: hidden;
+  @include euiBreakpoint('xs', 's') {
+    flex-grow: 0;
+    flex-wrap: wrap;
+    margin-left: 0;
+    margin-right: 0;
+    overflow-y: auto;
+  }
+}
+
+.dscApp__content {
+  flex-grow: 1;
+  flex-direction: column;
+  display: flex;
+  overflow: hidden;
```

Comment:
Since this is used in multiple places, it would be nice to create a mixin and use that one.

Reply:
thx! true, but I took a look and this css class is currently not in use (will be in the next iteration), so I've removed it